examples/quantum_annealing/price_optimization/QUBO-Pricing.ipynb

```diff
@@ -0,0 +1,2855 @@
+       "      <td>0</td>\n",
+       "      <td>0</td>\n",
+       "      <td>0</td>\n",
+       "      <td>...</td>\n",
+       "      <td>0</td>\n",
+       "      <td>0</td>\n",
+       "      <td>0</td>\n",
+       "      <td>0</td>\n",
+       "      <td>0</td>\n",
+       "      <td>0</td>\n",
+       "      <td>0</td>\n",
+       "      <td>0</td>\n",
+       "      <td>1</td>\n",
+       "      <td>-12589.992188</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>78</th>\n",
+       "      <td>0</td>\n",
+       "      <td>0</td>\n",
+       "      <td>0</td>\n",
+       "      <td>0</td>\n",
+       "      <td>1</td>\n",
+       "      <td>0</td>\n",
+       "      <td>0</td>\n",
+       "      <td>0</td>\n",
+       "      <td>0</td>\n",
+       "      <td>0</td>\n",
+       "      <td>...</td>\n",
+       "      <td>0</td>\n",
+       "      <td>0</td>\n",
+       "      <td>0</td>\n",
+       "      <td>0</td>\n",
+       "      <td>0</td>\n",
+       "      <td>0</td>\n",
+       "      <td>0</td>\n",
+       "      <td>0</td>\n",
+       "      <td>1</td>\n",
+       "      <td>-12555.671875</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "<p>5 rows × 50 columns</p>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "      X_000  X_001  X_002  X_003  X_004  X_005  X_006  X_007  X_008  X_009  \\\n",
+       "76        0      0      0      0      1      0      0      0      0      0   \n",
+       "914       0      0      1      0      0      0      0      0      0      1   \n",
+       "1331      0      0      0      0      1      0      0      0      0      0   \n",
+       "77        0      0      0      0      1      0      0      0      0      0   \n",
+       "78        0      0      0      0      1      0      0      0      0      0   \n",
+       "\n",
+       "      ...  X_040  X_041  X_042  X_043  X_044  X_045  X_046  X_047  X_048  \\\n",
+       "76    ...      0      1      0      0      0      0      0      0      1   \n",
+       "914   ...      0      1      0      0      0      0      0      0      1   \n",
+       "1331  ...      0      0      0      0      0      0      0      0      1   \n",
+       "77    ...      0      0      0      0      0      0      0      0      1   \n",
+       "78    ...      0      0      0      0      0      0      0      0      1   \n",
+       "\n",
+       "            energy  \n",
+       "76   -12752.164062  \n",
+       "914  -12602.234375  \n",
+       "1331 -12597.156250  \n",
+       "77   -12589.992188  \n",
+       "78   -12555.671875  \n",
+       "\n",
+       "[5 rows x 50 columns]"
+      ]
+     },
+     "execution_count": 15,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "## run dwave quantum annealing\n",
+    "num_shots = 10000\n",
+    "\n",
+    "sampler = BraketDWaveSampler(s3_folder,'arn:aws:braket:::device/qpu/d-wave/Advantage_system4')\n",
```

Comment:
`s3_folder` no longer required. Can be changed to
 `sampler = BraketDWaveSampler(device_arn='arn:aws:braket:::device/qpu/d-wave/Advantage_system4')`